File: Logitude/AngularModules/AngularModules/Infrastructure/EntityPMs/SharedUserQueryPM.ts

export class SharedUserQuery {
    private id: string;
    public get Id() { return this.id; }
    public set Id(newValue: string) { this.id = newValue; }
    
    private tenant: number;
    public get Tenant() { return this.tenant; }
    public set Tenant(newValue: number) { this.tenant = newValue; }

    private userId: string;
    public get UserId() { return this.userId; }
    public set UserId(newValue: string) { this.userId = newValue; }

    private queryId: string;
    public get QueryId() { return this.queryId; }
    public set QueryId(newValue: string) { this.queryId = newValue; }
}

File: Logitude/Logitude.BL/InfrastructureModel/EntityPMs/SharedUserQueryPM.cs
using Simplog.Server.Infrastructure;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Logitude.BL.InfrastructureModel.EntityPMs
{
    public class SharedUserQueryPM
    {
        [Key]
        public string Id { get; set; }
        public int Tenant { get; set; }        
        public string UserId { get; set; }
        public string QueryId { get; set; }

        public ChangeSetOperation ChangeSetOp { get; set; }
    }
}
